lib/osf-components/addon/components/registries/schema-block-group-renderer/template.hbs

```diff
@@ -0,0 +1,8 @@
+{{#each this.nonOptionBlocks as |block|}}
+    <Registries::SchemaBlockRenderer
+        @schemaBlock={{block}}
+        @changeset={{@changeset}}
+        @options={{this.optionBlocks}}
```

Comment:
Should this param be named `optionBlocks` instead of `options` to keep it consistent?